pkg/proxy/apis/config/validation/validation_test.go

```diff
@@ -282,14 +278,15 @@ func TestValidateKubeProxyConfiguration(t *testing.T) {
 					TCPCloseWaitTimeout:   &metav1.Duration{Duration: 5 * time.Second},
 				},
 			},
-			msg: "must be a valid CIDR block (e.g. 10.100.0.0/16 or fde4:8dba:82e1::/48)",
+			expectedErrs: field.ErrorList{field.Invalid(newPath.Child("ClusterCIDR"), "192.168.59.0", "must be a valid CIDR block (e.g. 10.100.0.0/16 or fde4:8dba:82e1::/48)")},
 		},
-		{
+		"Two ClusterCIDR addresses provided without IPv6 feature-enabled": {
```

Comment:
"dual stack" not "IPv6". (Single-stack IPv6 is available without any feature gate.)
(likewise below)